components/brave_ads/core/internal/creatives/creatives_builder.cc

```diff
@@ -104,6 +104,7 @@ CreativesInfo BuildCreatives(const CatalogInfo& catalog) {
 
         info.title = creative.payload.title;
         info.body = creative.payload.body;
+        info.embedding = creative_set.embedding;
```

Comment:
Please move to section above so these are in the same order as the data structure, thanks